iteration.qmd

```diff
@@ -322,21 +336,25 @@ df |>
     median = median(value, na.rm = TRUE),
     n_miss = sum(is.na(value))
   )
+
+# this feels unfinished, i.e. lacking the direct head-to-head comparison to something using across()
```

Comment:
In particular, I think there's another pivot operation that will happen and it would be get to really nail down that pattern, so the reader knows that's normal.